tikv/mvcc.go

```diff
@@ -131,7 +131,7 @@ func (store *MVCCStore) Prewrite(reqCtx *requestCtx, mutations []*kvrpcpb.Mutati
 	var buf []byte
 	var enc rowcodec.Encoder
 	for i, m := range mutations {
-		oldVal, err := store.checkPrewriteInDB(reqCtx, txn, m, startTS)
+		oldMeta, oldVal, err := store.checkPrewriteInDB(reqCtx, txn, m, startTS)
```

Comment:
Just a little curious, can we delay the vlog read to `Commit`, let `checkPrewriteInDB` just return `UserMeta`. So we can reduce some useless vlog read when `anyError == true`, and we don't need to store the old value in `LockStore`.

Reply:
Then if there is no error, there would be one more LSM read.

Reply:
Got.